### ISSUE TITLE
try to test multi xpu with example 

### ISSUE THREAD

**Issue by K-Alex13** [intel-analytics/ipex-llm#11091]
![image](https://github.com/intel-analytics/ipex-llm/assets/74948610/0a1a5d0d-0973-4db5-ab62-8934750ecd7f)
Due to the huggingface download problem, I download the model from following link.
https://huggingface.co/Qwen/Qwen1.5-14B-Chat/tree/main
Replace the model with the model's URL.  And the issue comes up.  Not sure what is going wrong Please help me.

**Comment by plusbang** [comment]
Hi, @K-Alex13 , if you have downloaded the model from https://huggingface.co/Qwen/Qwen1.5-14B-Chat/tree/main, please just replace `'Qwen/Qwen1.5-14B-Chat'` with your local model folder path here(https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/Deepspeed-AutoTP/run_qwen_14b_arc_2_card.sh#L38).

**Comment by K-Alex13** [comment]
Yes I already use this method, the error comes up after the process you mentioned

**Comment by K-Alex13** [comment]
And the error mentioned missing file are also not in the Qwen/Qwen1.5-14B-Chat files.

**Comment by plusbang** [comment]
> And the error mentioned missing file are also not in the Qwen/Qwen1.5-14B-Chat files.

If `model.safetensors.index.json` is not in your local folder, such error message would still occur. You may need to check whether all model files are available and complete in your local model folder.

**Comment by K-Alex13** [comment]
![image](https://github.com/intel-analytics/ipex-llm/assets/74948610/13eb1b79-ef37-402a-a925-75d0e8009b71)
what is the function of low-bit here.  I think it is 4 bit initial that the gpu needed will less than 16G so I do not know if this use two gpu here.  or can you please tell how to check the gpu usage during inference?

**Comment by K-Alex13** [comment]
![image](https://github.com/intel-analytics/ipex-llm/assets/74948610/81e5b4c8-6f11-4edc-94ed-453797ce59ad)
Why gpu 0 did not inference the results and gpu1 did.

**Comment by plusbang** [comment]
> what is the function of low-bit here. I think it is 4 bit initial that the gpu needed will less than 16G so I do not know if this use two gpu here. or can you please tell how to check the gpu usage during inference?

- As we introduced in [README](https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/Deepspeed-AutoTP/README.md), you could specify other low bit optimizations (such as `fp8`) through `--low-bit`.
- If you want to monitor GPU usage, you could use a tool named `xpu-smi`. Use `sudo apt install xpu-smi` to install, then you could use `sudo xpu-smi stats -d 0` to check memory usage of GPU 0.

> Why gpu 0 did not inference the results and gpu1 did.

- Both GPU did inference but we only print inference result of `RANK 0` [here](https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/Deepspeed-AutoTP/deepspeed_autotp.py#L128). In the log, `[0]` corresponds to output message of `RANK 0` while `[1]` is `RANK 1`.

**Comment by K-Alex13** [comment]
![image](https://github.com/intel-analytics/ipex-llm/assets/74948610/6bc62130-0444-4cfe-9b67-54138f8b4e8f)
still not working

**Comment by plusbang** [comment]
> ![image](https://private-user-images.githubusercontent.com/74948610/333013165-6bc62130-0444-4cfe-9b67-54138f8b4e8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MzI3MDcsIm5iZiI6MTcxNjQzMjQwNywicGF0aCI6Ii83NDk0ODYxMC8zMzMwMTMxNjUtNmJjNjIxMzAtMDQ0NC00Y2ZlLTliNjctNTQxMzhmOGI0ZThmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIzVDAyNDY0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjMjY3NDAyMTM5YjU0MThiZDk3MjgzZWJmNmZlYTRiNTJiMGNmOGY1MjVkZmJkNTNlNjhjODAyYWM0MDQzYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gfukwwJ96CSTJRlNPVt6uJ-fWlpEkeEf-6cJ1jHwRDU) still not working

According to your screenshot, maybe you could try `sudo apt install libmetee` and `sudo apt install libmetee-dev`.

**Comment by K-Alex13** [comment]
how to use them

**Comment by plusbang** [comment]
> how to use them

Sorry but I have no idea about the meaning of 'them'. ME TEE Library (libmetee/libmetee-dev) is a C library to access CSE/CSME/GSC firmware via, and `xpu-smi` tool seems to need. Could you use `xpu-smi` now?

**Comment by K-Alex13** [comment]
I install the packages you said above and try to us xpu-smi, same error comes up

**Comment by K-Alex13** [comment]
By the way I want to know if this is not a method which use two gpu as a bigger gpu to inference message.  It just put model in two different gpu separately and inference separately?

**Comment by plusbang** [comment]
> I install the packages you said above and try to us xpu-smi, same error comes up

Maybe you could try these steps?
```bash
sudo apt-get autoremove libmetee-dev
sudo apt-get autoremove libmetee
sudo apt-get install libmetee
sudo apt-get install libmetee-dev
sudo apt-get install xpu-smi
```

> By the way I want to know if this is not a method which use two gpu as a bigger gpu to inference message. It just put model in two different gpu separately and inference separately?

The model is separated and put to two GPUs, each GPU need less memory to inference. In this way, you could treat two GPUs as a bigger one.
